lib/rubocop/cop/salsify/rspec_string_literals.rb

```diff
@@ -14,6 +14,7 @@ module Salsify
       class RspecStringLiterals < RuboCop::Cop::RSpec::Base
         extend RuboCop::Cop::AutoCorrector
         include ConfigurableEnforcedStyle
+        include StringHelp
```

Comment:
This one was funny, StringHelp extracted the actual node visitor from the previous StringLiteralHelp, so this cop suddenly was completely broken (visited zero nodes). This restores the `on_str` handler.